text_mining.py

```diff
@@ -0,0 +1,131 @@
+import requests
```

Comment:
Include a header comment with your name and brief description of what this file does.